src/game_api/search.cpp

```diff
@@ -1745,6 +1745,13 @@ std::unordered_map<std::string_view, AddressRule> g_address_rules{
             .at_exe()
             .function_start(),
     },
+    {
+        "play_sound"sv,
```

Comment:
This is a different than `construct_soundposition_ptr` and I'm not sure why, the game uses this function for all simple short sounds, and for everything that needs the position to be tracked it uses `construct_soundposition_ptr` and in most cases it's stored somewhere in entity memory. But both seam to make more or less the same thing, creating `SoundPosition` (now `SoundMeta`)

Reply:
next issue, how to use the `VANILLA_SOUND` enum, as that contains strings and this uses ids, should I just dump the data to some vector/map when it's makes the enum or something?